### PR TITLE
killed mutants

### PR DESCRIPTION
**Pull request by pamparam83**
killed mutants in `DateTimeHandler`

### CODE REVIEW

File: src/Rule/DateTimeHandler.php [hunk 1]
```diff
@@ -32,13 +32,23 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
...
 
+        $errors = \DateTime::getLastErrors();
         // Before PHP 8.2 may return array instead of false (see https://github.com/php/php-src/issues/9431).
-        $errors = \DateTime::getLastErrors() ?: [ 'error_count' => 0, 'warning_count' => 0 ];
-        if ($errors['error_count'] !== 0  ||  $errors['warning_count'] !== 0) {
-            $result->addError($rule->getMessage(), [
-                'attribute' => $context->getTranslatedAttribute(),
-                'value' => $value,
-            ]);
+        /** @infection-ignore-all */
```
Comment (vjik) [inline src/Rule/DateTimeHandler.php:37]: Can you add tests instead of ignore?

Comment (vjik) [reply src/Rule/DateTimeHandler.php]: I make a little refactoring. It is help remove mutants. If this OK for you, then I will merge this PR.

**Comment by pamparam83** [comment]
@vjik I checked at php8.2
3 mutants left

File: src/Rule/DateTimeHandler.php (hunk 1 above)
Comment (pamparam83) [reply src/Rule/DateTimeHandler.php]: if you need to kill only in version 81, then ok.

Comment (vjik) [reply src/Rule/DateTimeHandler.php]: Good catch 👍. I changed version for mutation testing to 8.3. And fix mutants.

**Review by vjik — APPROVED** [review]

**Pull request merged**